### PR TITLE
Bugfix/error collector multiple instances

### PR DESCRIPTION
**Pull request by rcordeau**
This change is [Reviewable](https://reviewable.io/reviews/smarterapp/rdw_ingest/94)

### CODE REVIEW

File: package-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/utils/ParserHelper.java [hunk 1]
```diff
@@ -11,12 +11,12 @@
 /**
  * Helper methods for parsing
  */
-@Component
```
Comment (malaffoon) [inline package-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/utils/ParserHelper.java]: oops

File: package-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentParserIT.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+@Transactional
+@CachingTest
+@ContextConfiguration(classes = {PackageProcessorApplication.class})
+public class DefaultAssessmentParserIT {
+
+    @Autowired
+    private DefaultAssessmentParser assessmentParser;
+
+
+    // naming convention of itShould<full sentence in camel case>()
```
Comment (malaffoon) [inline package-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentParserIT.java]: remove obsolete comment?

File: package-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentParserIT.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        assertThat(asmtSet.isEmpty());
+        try {
+            final Iterable<CSVRecord> records =
+                    CSVFormat.RFC4180.withFirstRecordAsHeader().parse(new InputStreamReader(this.getClass().getResourceAsStream("/IAB_English-bad.items.csv")));
+
+            for (CSVRecord record : records) {
+                asmtSet.add(assessmentParser.parse(record, errorCollector));
+            }
+        }catch(IOException ioe) {
+
```
Comment (malaffoon) [inline package-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentParserIT.java]: if i see empty catch block in a test, i usually stick in a `fail(ioe.getMessage())`

File: package-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/utils/ParserHelper.java [hunk 4]
```diff
@@ -11,12 +11,12 @@
...
+//@Component
```
Comment (jtreuting) [inline package-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/utils/ParserHelper.java]: Can you delete since it's not needed.

Comment (rcordeau) [inline package-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/utils/ParserHelper.java]: Oops. Thought I had. Will do.

File: package-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentParserIT.java (hunk 2 above)
Comment (rcordeau) [inline package-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentParserIT.java]: Will do

File: package-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentParserIT.java (hunk 3 above)
Comment (rcordeau) [inline package-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultAssessmentParserIT.java]: I'll add it to the signature

**Pull request merged**